app/hub/Views/UpgradePlanView/UpgradePlanView.jsx

```diff
@@ -0,0 +1,958 @@
+/**
+ * Upgrade Plan View Component
+ *
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2019 Ghostery, Inc. All rights reserved.
```

Comment:
Might as well make this 2020 off the bat, Tino auto-updates this periodically but since we're in the code anyway

Reply:
Changed to 2020!